src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -66,11 +66,16 @@ public class ConvertFromMarkdownCommand : PSCmdlet
         /// </summary>
         protected override void BeginProcessing()
         {
-            _mdOption = this.CommandInfo.Module.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
-
-            if (_mdOption == null)
+            // If we have the moduleInfo then store are module scope variable
+            if (this.CommandInfo.Module != null)
+            {
+                _mdOption = this.CommandInfo.Module?.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
```

Comment:
You have checked `ComamndInfo.Module != null`, so no need to use null-conditional operator here.

Reply:
Fixed with refactor